stable/yugaware/templates/configs.yaml

```diff
@@ -108,11 +108,7 @@ data:
       # CORS config
       cors {
         pathPrefixes = ["/"]
-{{- if .Values.tls.enabled }}
-        allowedOrigins = ["https://{{ .Values.tls.hostname }}"]
```

Comment:
Why removing the tls hostname? We need that + the teleport hostname used when proxying platform UIs

Reply:
I don't remove it, just put to helpers to include there.
As Teleport Platform URL is dynamic (depends on cloud + network_id + region + platform_id), so I can't add it here.